modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc

```diff
@@ -8,10 +8,45 @@
 
 .Procedure 
 
-. Change the current directory to the front-end directory:
+ifeval::["{context}" == "creating-a-single-component-application-with-odo"]
```

Comment:
We've switched to using the ifeval statements to set variables based on the context statements and then just use if statements to conditionalize the text, like here: https://raw.githubusercontent.com/openshift/openshift-docs/master/modules/installation-initializing.adoc

Reply:
Oh, I missed that change! Sorry, fixed. The build looks fine, so, hopefully, I've done in correctly. 
What was the reasoning for switching to variables?

Reply:
You can't put two arguments in an ifeval statement, so if you need to use the same text in multiple locations, you have to duplicate it. It's a good way to make it easier to reuse content as you use it in more locations.

Reply:
Thank you! That makes total sense.